crawl-ref/source/mon-data.h

```diff
@@ -2132,15 +2132,41 @@ DUMMY(MONS_SPIDER, 's', CYAN, "spider", TILEP_MONS_REDBACK)
     MONS_ORB_SPIDER, 's', MAGENTA, "orb spider",
     M_NO_SKELETON | M_MAINTAIN_RANGE,
     MR_VUL_POISON,
-    20, MONS_SPIDER, MONS_ORB_SPIDER, MH_NATURAL, 40,
-    { {AT_BITE, AF_PLAIN, 5}, AT_NO_ATK, AT_NO_ATK, AT_NO_ATK },
-    7, 420,
+    12, MONS_SPIDER, MONS_ORB_SPIDER, MH_NATURAL, 40,
+    { {AT_BITE, AF_PLAIN, 15}, AT_NO_ATK, AT_NO_ATK, AT_NO_ATK },
+    10, 420,
     3, 10, MST_ORB_SPIDER, true, S_HISS,
     I_ANIMAL, HT_LAND, 12, SPELL_ENERGY(20),
     MONUSE_NOTHING, SIZE_SMALL, MON_SHAPE_ARACHNID,
     {TILEP_MONS_ORB_SPIDER}, TILE_CORPSE_ORB_SPIDER
 },
 
+{
+    MONS_BROODMOTHER, 's', LIGHTCYAN, "broodmother",
+    M_NO_SKELETON,
+    MR_NO_FLAGS,
+    10, MONS_SPIDER, MONS_BROODMOTHER, MH_NATURAL, 60,
+    { {AT_BITE, AF_SPIDER, 45}, AT_NO_ATK, AT_NO_ATK, AT_NO_ATK },
```

Comment:
Wow, that's one big bite! Vaguely wonder about tuning it down slightly, maybe to 40?